compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/model/symbols/BResourceFunction.java

```diff
@@ -34,23 +37,47 @@ public class BResourceFunction extends BAttachedFunction {
     public BVarSymbol restPathParam;
     public List<Name> resourcePath;
     public Name accessor;
+    public BTupleType resourcePathType;
```

Comment:
Can we get the default accessor "get" here as a static final member, may be called "DEFAULT_ACCESSOR"?